api/v1alpha1/openapi.yaml

```diff
@@ -775,6 +775,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/EnrollmentRequest'
+        "208":
```

Comment:
208 looking at the docs it seems specific to a 207 response in WebDav. Is this a non fatal conflict so 409?

- https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/208

Reply:
May be 409, right. As long as the agent understands what the server means with that.

It could also simply be 200 (OK) ... because we are not creating anything new (202) and we don't really imply there is an error, and we are returning the object, and not an error-type response.

WDYT?

Reply:
anything but 500 works for me :)

Reply:
Let's go with that the 208 then which is meaningful for our server<-> agent context, and  I will follow up rethinking how we handle those contexts. 

Thank you!